README.md

```diff
@@ -51,7 +51,7 @@ Then configure the rules you want to use under the rules section.
 - [`jsx-classname-namespace`](docs/rules/jsx-classname-namespace.md): Ensure JSX className adheres to CSS namespace guidelines
 - [`jsx-gridicon-size`](docs/rules/jsx-gridicon-size.md): Enforce recommended Gridicon size attributes
 - [`no-lodash-import`](docs/rules/no-lodash-import.md): Disallow importing from the root Lodash module
-
+- [`no-this-translate`](docs/rules/no-this-translate.md): Disallow this.translate()
```

Comment:
We need a line break after this line, otherwise the following heading is collapsed.